### DISCUSSION TITLE
Installation process does not continue

### DISCUSSION THREAD

**Discussion by SaschaNe** [Laravel-Backpack/community-forum#688] [Q&A (Help)] [answered]
Hey, I'm running the command "sudo php artisan backpack:install" but while creating the users table, it does not continue.
Laravael contains already Jetstream and Socialstream.

 STEP 1  Installing Backpack CRUD.
  Publishing configs, views, routes and assets ............................................ DONE
  Creating users table ................................................................. RUNNING

I've already tried it multiple times, and it just won't continue and shows me "RUNNING" for hours. The database connection is available and running well.

## PHP Version:
PHP 8.2.10 (cli) (built: Sep  2 2023 06:59:22) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.10, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.10, Copyright (c), by Zend Technologies

## Composer.json:
    "require": {
        "php": "^8.1",
        "backpack/crud": "^6.2",
        "doctrine/dbal": "^3.6",
        "geoip2/geoip2": "^2.13",
        "guzzlehttp/guzzle": "^7.2",
        "joelbutcher/socialstream": "^4.4",
        "laravel/framework": "^10.10",
        "laravel/jetstream": "^4.0",
        "laravel/sanctum": "^3.2",
        "laravel/tinker": "^2.8",
        "livewire/livewire": "^3.0",
        "rappasoft/laravel-authentication-log": "^3.0",
        "torann/geoip": "^3.0"
    },

**Comment by tabacitu** [comment, marked as answer]
Hi @SaschaNe ,

If the automatic installation isn't working for you, for any reason, you can follow the [manual installation process](https://backpackforlaravel.com/docs/6.x/crud-how-to#how-to-manually-install-backpack). If you already have the `users` table then indeed you don't need to run the migrations, so you can skip that step entirely.

Let me know if that works for you, or you get a roadblock in the manual installation too.

Cheers!

**Reply by tabacitu** [reply]
Note to myself and the team: I did investigate _why_ this might have happened, but I have no idea. I tried installing Backpack on a fresh project with Laravel Jetstream and it installed fine for me:
<img width="1510" alt="CleanShot 2023-09-26 at 11 30 18@2x" src="https://github.com/Laravel-Backpack/community-forum/assets/1032474/b38067d4-0b51-4a59-bcdd-dc645b5a200c">

**Reply by SaschaNe** [reply]
Thank you, I could continue with the manual process. 

I received the following errors while doing this:
```
sudo php artisan vendor:publish --provider="Backpack\CRUD\BackpackServiceProvider" --tag="minimum"
   INFO  Publishing [minimum] assets.  
  Copying directory [vendor/backpack/crud/src/config] to [config] ........................ DONE
  File [resources/views/vendor/backpack/ui/inc/menu_items.blade.php] already exists ... SKIPPED  
   ERROR  Can't locate path: <vendor/backpack/crud/src/public>.  
  File [routes/backpack/custom.php] already exists .................................... SKIPPED  
  Copying directory [vendor/creativeorange/gravatar/config] to [config] .................. DONE
```

The users aren't created automatically in the database. While registering as a new admin user, the process is also not possible. It does also not show any error in the front end or log.

**Reply by SaschaNe** [reply]
I used the automated process now, it worked @tabacitu 
The problem was custom columns in the table "users", which had no default value.

The next problem I'm facing is to install the add-ons. 
```
Require Backpack DevTools on dev ................................................................. ERROR
  Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - backpack/devtools[2.0.0, ..., 2.0.3] require livewire/livewire ^2.4 -> found livewire/livewire[v2.4.0, ..., v2.12.6] but it conflicts with your root composer.json require (^3.0).
    - backpack/devtools[1.0.35, ..., 1.3.5] require backpack/crud ^5.3 -> found backpack/crud[5.3.0, ..., 5.6.1] but it conflicts with your root composer.json require (^6.2).
    - backpack/devtools[1.0.8, ..., 1.0.28] require backpack/crud ^4.1.49 -> found backpack/crud[4.1.49, ..., 4.1.71] but it conflicts with your root composer.json require (^6.2).
    - backpack/devtools[1.0.2, ..., 1.0.7] require backpack/generators ^3.1.9 -> found backpack/generators[v3.1.9, ..., v3.3.17] but it conflicts with your root composer.json require (^4.0).
    - backpack/devtools[1.0.29, ..., 1.0.34] require backpack/crud ^4.1.49|^5.0 -> found backpack/crud[4.1.49, ..., 4.1.71, 5.0.0, ..., 5.6.1] but it conflicts with your root composer.json require (^6.2).
    - backpack/devtools[1.0.0, ..., 1.0.1] require backpack/generators ^3.0 -> found backpack/generators[3.0.0, ..., v3.3.17] but it conflicts with your root composer.json require (^4.0).
    - Root composer.json requires backpack/devtools * -> satisfiable by backpack/devtools[1.0.0, ..., 1.3.5, 2.0.0, 2.0.1, 2.0.2, 2.0.3].
```

How can I solve it without to downgrade?

**Reply by tabacitu** [reply]
Hey @SaschaNe - indeed, DevTools doesn't support Livewire v3 yet, it's still using Livewire v2. We're working on adding support for it as we speak. Happy to refund DevTools if you need it _right now_, it will probably take us 1-2 weeks more to add Livewire v3 support.
